move/.gitignore

```diff
@@ -0,0 +1 @@
+wandb/
```

Comment:
Maybe you can remove this .gitignore, since the one at the root will take care of ignoring?